doc/sphinx/proof-engine/tactics.rst

```diff
@@ -2804,14 +2804,15 @@ the conversion in hypotheses :n:`{+ @ident}`.
 
    This tactic applies to a goal that has the form::
 
-     forall (x:T1) ... (xk:Tk), t
+     forall (x:T1) ... (xk:Tk), T
 
-   with :g:`t` :math:`\beta`:math:`\iota`:math:`\zeta`-reducing to :g:`c t`:sub:`1` :g:`... t`:sub:`n` and :g:`c` a
+   with :g:`T` :math:`\beta`:math:`\iota`:math:`\zeta`-reducing to :g:`c t`:sub:`1` :g:`... t`:sub:`n` and :g:`c` a
    constant. If :g:`c` is transparent then it replaces :g:`c` with its
    definition (say :g:`t`) and then reduces
    :g:`(t t`:sub:`1` :g:`... t`:sub:`n` :g:`)` according to :math:`\beta`:math:`\iota`:math:`\zeta`-reduction rules.
 
 .. exn:: Not reducible.
+.. exn:: No head constant to reduce.
```

Comment:
Please had `:undocumented:` to these two error messages: see https://github.com/coq/coq/tree/master/doc/sphinx#coq-objects and #7602.